### ISSUE TITLE
Allow batching of timers or metrics

### ISSUE THREAD

**Issue by nicjansma** [akamai/mpulse.js#5]
For example, if an API consumer wants to send several timers all at the same time, instead of on multiple beacons, we could have:

`sendTimers({"a": 1, "b": 2})`

(instead of this, which would send several beacons):

```
sendTimer("a", 1);
sendTimer("b", 2);
```

We could also combine metrics and timers:

```
send({
  "metrics": {"m1": 1},
  "timers": {"a": 1, "b": 2},
});
```

**Comment by akamaimike** [comment]
Thanks for reporting this issue. We will be closing this issue due to inactivity. Please feel free to open a new issue if needed. Kind regards, Mike

**Comment by bluesmoon** [comment]
Hi @akamaimike, I don't know all the details, but perhaps this shouldn't apply to issues that were opened by the project maintainer for tracking.

**Comment by akamaimike** [comment]
Good point, we are looking into cleaning up the open issues on GitHub. We might be moving these enhancements requests to Jira instead but we are still in talks with the different engineering teams.

**Comment by David-Development** [comment]
Are there any updates on this feature request? I think it would be even more useful if you could not only batch timers and metrics yourself but also introduce timely batching as proposed in https://github.com/akamai/mpulse.js/issues/6. This would be extremely useful to us as we want to track certain features of our website which might produce quite a few beacons.
